src/Workspaces/Core/Portable/Workspace/Workspace_Events.cs

```diff
@@ -51,7 +55,7 @@ protected Task RaiseWorkspaceChangedEventAsync(WorkspaceChangeKind kind, Solutio
                 projectId = documentId.ProjectId;
             }
 
-            var ev = _eventMap.GetEventHandlers<EventHandler<WorkspaceChangeEventArgs>>(WorkspaceChangeEventName);
+            var ev = GetEventHandlers<WorkspaceChangeEventArgs>(WorkspaceChangeEventName);
```

Comment:
Should this be moved into _eventMap itself somehow so we don't risk somebody calling _eventMap.GetEventHandlers directly and screwing this up?

Reply:
we could, but eventmap is not tied to workspace so.. not sure we want to tightly couple these 2. if one wants to, I guess we could create extension method but, I dont think I am going to do that in this PR.